test/circle/install.sh

```diff
@@ -32,6 +32,14 @@ helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.c
 # Pinning to a version for consistency
 sudo pip install yamllint==1.8.1
 
+# Install a Yamale. yaml schema validator
+# Pinning to a version for consistency
+sudo pip install yamale==1.7.0
+
+# Install a yq bash yaml cli tool
+# Pinning to a version for consistency
+sudo pip install yq==2.1.1
```

Comment:
The tool installed below (`https://github.com/mikefarah/yaml`) is also a yaml query tool. It's been renamed yq (we're still using an older version). We should install one yaml querying tool and use that consistently.

I'm indifferent to the one we use.